common/source/docs/common-escs-and-motors.rst

```diff
@@ -29,19 +29,45 @@ Motors
 [/site]
 
 
-ESC 
-====
+ESC for Brushless Motors 
+========================
+
+ESCs (Electronic Speed Controls) have many different protocols for communicating with the autopilot for motor speed control, and and can also provide telemetry information on motor rpm, battery voltage. The
+capabilities will vary with each individual ESC model. Some ESCs have specialized firmware that allows easy configuration of operating parameters. The following sections
+explain the required ArduPilot setup to utilize the protocols, telemetry, and setup programs that various ESCs utilize.
+See :ref:`common-esc-guide` for a guide to terminology.
+
+Protocols
+---------
+
 .. toctree::
     :maxdepth: 1
 
-    BLHeli/BLHeli32 Capable ESCs <common-blheli32-passthru>
-    Currawong Velocity ESCs <common-velocity-can-escs>
-    DroneCAN ESCs <common-uavcan-escs>
+    PWM, OneShot and OneShot125 ESCs <common-brushless-escs>
     DShot ESCs <common-dshot-escs>
+    DroneCAN ESCs <common-uavcan-escs>
+    Currawong Velocity ESCs <common-velocity-can-escs>
     FETtec OneWire ESCs <common-fettec-onewire>
-    PWM, OneShot and OneShot125 ESCs <common-brushless-escs>
     KDE CAN ESCs <common-kde-can-escs>
-    ESC Grounding and Wiring Considerations<common-esc-grounding>
+
+
+ESCs using BLHeli32 or BLHeli-S Configuration Firmware
+------------------------------------------------------
+
+.. toctree::
+    :maxdepth: 1
+
+    BLHeli/BLHeli32 Capable ESCs <common-blheli32-passthru>
+
+
+Telemetry
+---------
+
+.. toctree::
+    :maxdepth: 1
+
+    ESC Telemetry <common-esc-telemetry>
```

Comment:
This should probably be BLHeli ESC Telemetry

Reply:
As stated for the reasons above, I disagree...but will add a note about current ARduPilot implementations being only BLHeli ESCs